 /bin/install -c -m 644 README default.svg A4_landscape.svg A4.svg black_opaque.svg white_opaque.svg business_card_85x54mm.svg business_card_90x50mm.svg CD_cover_300dpi.svg DVD_cover_regular_300dpi.svg DVD_cover_slim_300dpi.svg DVD_cover_superslim_300dpi.svg DVD_cover_ultraslim_300dpi.svg default.svg default.ca.svg default.cs.svg default.de.svg default.eo.svg default.eu.svg default.es.svg default.fi.svg default.fr.svg default.hu.svg default.it.svg default.lt.svg default.pl.svg default.pt_BR.svg default.sk.svg default_mm.svg default_pt.svg desktop_1024x768.svg desktop_1600x1200.svg desktop_640x480.svg desktop_800x600.svg fontforge_glyph.svg icon_16x16.svg icon_32x32.svg icon_48x48.svg icon_64x64.svg Letter_landscape.svg '/var/tmp/fst/pkg/usr/share/inkscape/templates'
/bin/install: will not overwrite just-created `/var/tmp/fst/pkg/usr/share/inkscape/templates/default.svg' with `default.svg'
make[3]: *** [install-templatesDATA] Error 1
make[3]: Leaving directory `/var/tmp/fst/src/inkscape-0.46/share/templates'
make[2]: *** [install-am] Error 2
make[2]: Leaving directory `/var/tmp/fst/src/inkscape-0.46/share/templates'
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory `/var/tmp/fst/src/inkscape-0.46/share'
make: *** [install-recursive] Error 1

diff --git a/share/templates/Makefile.am b/share/templates/Makefile.am
index 62ee5a1..bbd320c 100644
--- a/share/templates/Makefile.am
+++ b/share/templates/Makefile.am
@@ -15,7 +15,6 @@ templates_DATA = \
 	DVD_cover_slim_300dpi.svg \
 	DVD_cover_superslim_300dpi.svg \
 	DVD_cover_ultraslim_300dpi.svg \
-	default.svg \
 	default.ca.svg \
 	default.cs.svg \
 	default.de.svg \
